actionview/lib/action_view/layouts.rb

```diff
@@ -405,11 +405,11 @@ def _normalize_layout(value)
     #
     # ==== Parameters
     # * <tt>formats</tt> - The formats accepted to this layout
-    # * <tt>require_layout</tt> - If set to true and layout is not found,
-    #   an +ArgumentError+ exception is raised (defaults to false)
+    # * <tt>require_layout</tt> - If set to +true+ and layout is not found,
+    #   an +ArgumentError+ exception is raised (defaults to +false+)
```

Comment:
@fxn there are loads of places that use `+true+` in the docs but will accept a truthy value. Also whether it's true or +true+ is a very subtle way of indicating that difference to users who are likely to be novices. Perhaps we need to come up with a clearer indication of the difference?